eng/build.yml

```diff
@@ -1,53 +1,53 @@
 parameters:
+  # Agent OS identifier and used as job name
   agentOs: ''
-  queue: {}
+
+  # Agent pool
+  pool: {}
+
+  # Additional variables
   variables: {}
 
-phases:
+  # Build strategy - matrix
+  strategy: {}
+
+  # Job timeout
+  timeoutInMinutes: 180
+
+jobs:
 - template: /eng/common/templates/phases/base.yml
   parameters:
     name: ${{ parameters.agentOs }}
+    ${{ if eq(parameters.agentOs, 'Windows_NT') }}:
```

Comment:
Technically, this condition shouldn't be needed.  The templates should do the right thing